--- nnn-2.1/Makefile~	2018-11-23 12:22:14.000000000 +0100
+++ nnn-2.1/Makefile	2018-11-25 17:45:26.122311818 +0100
@@ -30,7 +30,7 @@
 	$(RM) $(DESTDIR)$(DESKTOPICONPREFIX)/scalable/apps/nnn.svg
 	$(RM) $(DESTDIR)$(DESKTOPICONPREFIX)/64x64/apps/nnn.png
 
-install: all
+install:
 	$(INSTALL) -m 0755 -d $(DESTDIR)$(PREFIX)/bin
 	$(INSTALL) -m 0755 $(BIN) $(DESTDIR)$(PREFIX)/bin
 	$(INSTALL) -m 0755 -d $(DESTDIR)$(MANPREFIX)/man1
