--- nnn-3.3/Makefile.orig	2020-08-06 12:43:07.917453708 +0200
+++ nnn-3.3/Makefile	2020-08-06 12:46:43.682792814 +0200
@@ -41,7 +41,7 @@
 endif
 
 ifeq ($(strip $(O_PCRE)),1)
-	CPPFLAGS += -DPCRE
+	override CPPFLAGS += -DPCRE
 	LDLIBS += -lpcre
 endif
 
@@ -62,7 +62,7 @@
 endif
 
 ifeq ($(strip $(O_CTX8)),1)
-	CPPFLAGS += -DCTX8
+	override CPPFLAGS += -DCTX8
 endif
 
 ifeq ($(strip $(O_ICONS)),1)
@@ -75,9 +75,9 @@
 	LDLIBS_CURSES ?= -lncurses
 endif
 
-CFLAGS += -std=c11 -Wall -Wextra -Wshadow
-CFLAGS += $(CFLAGS_OPTIMIZATION)
-CFLAGS += $(CFLAGS_CURSES)
+override CFLAGS += -std=c11 -Wall -Wextra -Wshadow
+override CFLAGS += $(CFLAGS_OPTIMIZATION)
+override CFLAGS += $(CFLAGS_CURSES)
 
 LDLIBS += $(LDLIBS_CURSES)
 
